showcase/src/assets/examples/date-picker/weekends.html

```diff
@@ -0,0 +1,10 @@
+<stark-date-picker 
+    pickerId="picker-only-weekends"
+    pickerName="only-weekends"
+    [date]="currentDate" 
+    [dateFilter]="'OnlyWeekends'" 
+    label="Choisir une date" 
```

Comment:
I think all our example texts should be in English. What do you think about this ?